app/budget/directives/modals/budgetConfig/instructorCostConfig/instructorCostConfig.html

```diff
@@ -2,14 +2,21 @@
 		<div class="instructor-cost-config__instructor" ng-repeat="instructor in instructors | orderBy: 'lastName'">
 			<div class="instructor-cost-config__input">
 				<ipa-input mode="'currency'"
-					value="instructor.cost"
-					on-update="updateInstructorCost(instructor)">
+					value="instructor.instructorCost.cost"
+					on-update="updateInstructorCost(instructor.instructorCost)">
 				</ipa-input>
 			</div>
 
+			<div class="instructor-cost-config__type-container">
+				<instructor-type-dropdown instructor-types="instructorTypes"
```

Comment:
You forgot to close the opening tag, e.g. the >

Reply:
This one concerned me for a second as I thought my local and server copy were disagreeing. the line is just WAY to the right due to how github is rendering the tabs. I'll rewrite this in the mixed tab/spaces style that works better, I've been going through and fixing that as I see them.